homeworks/html-css/Viktor Patraboi/index.html

```diff
@@ -0,0 +1,58 @@
+<html>
```

Comment:
Do not use spaces in folder's name

Reply:
You had to use your GitHub's username as folder's name, which is @okedo.

Reply:
resolved